l1-contracts/test/foundry/l1/integration/L1GatewayTests.t.sol

```diff
@@ -66,6 +66,7 @@ contract L1GatewayTests is L1ContractDeployer, ZKChainDeployer, TokenDeployer, L
         }
     }
 
+    /*
```

Comment:
why was this function commented out?

Reply:
Uncommented them right now. Earlier these tests were failing randomly (maybe a cache issue).